zntrack/cli/cli.py

```
@@ -80,6 +152,19 @@ def run(
+
+    # If node_path doesn't contain a dot, treat it as a stage name from dvc.yaml
+    if "." not in node_path:
+        try:
+            parsed_node_path, parsed_name = parse_command_from_dvc_yaml(node_path)
+            # Use parsed values if not overridden by command line arguments
+            if name is None:
+                name = parsed_name
+            node_path = parsed_node_path
+        except (FileNotFoundError, ValueError) as e:
+            typer.echo(f"Error: {e}", err=True)
+            raise typer.Exit(1)
```

Ruff (0.14.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)